### PR TITLE
[DONE][2.1.1][MERGE ME][Doc] Russian Translation

### PR DESCRIPTION
**Pull request by gbezyuk**
Don't merge this PR yet, it's a work in progress.

Translation progress:

- [x] README.md
- [x] SUMMARY.md

- [x] installation.md
- Essentials
  - [x] getting-started.md
  - [x] dynamic-matching.md
  - [x] nested-routes.md
  - [x] navigation.md
  - [x] named-routes.md
  - [x] named-views.md
  - [x] redirect-and-alias.md
  - [x] history-mode.md
- Advanced
  - [x] navigation-guards.md
  - [x] meta.md
  - [x] transitions.md
  - [x] data-fetching.md
  - [x] scroll-behavior.md
  - [x] lazy-loading.md
- API Reference
  - [x] router-link.md
  - [x] router-view.md
  - [x] route-object.md
  - [x] options.md
  - [x] router-instance.md
  - [x] component-injections.md

Proof-reading progress:

- [x] README.md
- [x] SUMMARY.md

- [x] installation.md
- Essentials
  - [x] getting-started.md
  - [x] dynamic-matching.md
  - [x] nested-routes.md
  - [x] navigation.md
  - [x] named-routes.md
  - [x] named-views.md
  - [x] redirect-and-alias.md
  - [x] history-mode.md
- Advanced
  - [x] navigation-guards.md
  - [x] meta.md
  - [x] transitions.md
  - [x] data-fetching.md
  - [x] scroll-behavior.md
  - [x] lazy-loading.md
- API Reference
  - [x] router-link.md
  - [x] router-view.md
  - [x] route-object.md
  - [x] options.md
  - [x] router-instance.md
  - [x] component-injections.md

### CODE REVIEW

**Comment by gbezyuk** [comment]
Currently, only API reference is left to translate. Yet everything is still subject for proof-reading.

File: docs/ru/advanced/scroll-behavior.md [hunk 1]
```diff
@@ -58,4 +58,4 @@ scrollBehavior (to, from, savedPosition) {
 }
 ```
 
-We can also use [route meta fields](meta.md) to implement fine-grained scroll behavior control. Check out a full example [here](https://github.com/vuejs/vue-router/blob/dev/examples/scroll-behavior/app.js).
+Можно также использовать [метаданыне путей](meta.md) для более сложного управления скроллингом. Полная реализация подхода содержится в [этом примере](https://github.com/vuejs/vue-router/blob/dev/examples/scroll-behavior/app.js).
```
Comment (akost) [inline docs/ru/advanced/scroll-behavior.md]: Typo: "метаданыне"
Должно быть "метаданные".

**Comment by gbezyuk** [comment]
Done, ready to be merged

**Comment by gbezyuk** [comment]
@yyx990803 seems the site didn't update after this PR merged. Could you please check?

**Pull request merged**